grafana-alloy.yaml

```diff
@@ -12,6 +12,9 @@ environment:
       - busybox
       - ca-certificates-bundle
       - go
+      - nodejs
```

Comment:
should we pin nodejs to a major version?

Reply:
they seem using nodejs-18: https://github.com/grafana/alloy/blob/a014bda1725d7817d34523f956812d3ae5e0c809/tools/build-image/Dockerfile#L71-L72

Reply:
```
Status: Downloaded newer image for grafana/alloy-build-image:v0.1.3
root@0ef59700d4da:/# node --version
v18.20.4
root@0ef59700d4da:/#
```

lets use nodejs-18